src/server/auth/server/api_server.go

```diff
@@ -475,55 +354,10 @@ func (a *apiServer) Activate(ctx context.Context, req *auth.ActivateRequest) (re
 func (a *apiServer) Deactivate(ctx context.Context, req *auth.DeactivateRequest) (resp *auth.DeactivateResponse, retErr error) {
 	a.LogReq(req)
 	defer func(start time.Time) { a.LogResp(req, resp, retErr, time.Since(start)) }(time.Now())
-	if !a.isActive() {
-		// users should be able to call "deactivate" from the "partial" activation
-		// state, in case activation fails and they need to revert to "none"
-
-		// TODO: we should consider if we want to allow deactivation to proceed
-		// even if the state is "none", in case they're in a bad state
-		return nil, auth.ErrNotActivated
-	}
-
-	// Check if the cluster is in a partially-activated state. If so, allow it to
-	// be completely deactivated so that it returns to normal
-	var ppsUserIsAdmin bool
```

Comment:
I missed this when I removed partial activation. There's no auth logic in `Deactivate` anymore, it's all handled in the interceptor.